src/theme.scss

```diff
@@ -87,7 +87,7 @@ $button-background-active: rgba(255, 255, 255, 0.2);
 
   background: $button-background;
   border-style: solid;
-  border-width: 1px;
+  border-width: 2px;
```

Comment:
do you think it would be better to have a variable for the `border-width`? since it is repeating itself multiple times